lib/src/clojure_lsp/crawler.clj

```diff
@@ -11,14 +11,60 @@
    [clojure.java.io :as io]
    [clojure.string :as string]
    [lsp4clj.protocols.logger :as logger]
-   [lsp4clj.protocols.producer :as producer])
+   [lsp4clj.protocols.producer :as producer]
+   [medley.core :as medley])
   (:import
    (java.net URI)))
 
 (set! *warn-on-reflection* true)
 
 (def startup-logger-tag "[Startup]")
 
+(defn lerp "Linear interpolation" [a b t] (+ a (* (- b a) t)))
+
+(defn init-tasks [tasks]
+  (->> tasks
+       (partition-all 2 1)
+       (map (fn [[task next-task]]
+              (assoc task :task/end-percent (if next-task
+                                              (dec (:task/start-percent next-task))
+                                              100))))
+       (medley/index-by :task/id)))
+
+(def fast-tasks
+  (init-tasks
+    [{:task/start-percent   0, :task/title "clojure-lsp",             :task/id :start}
```

Comment:
I used to really hate the map namespace syntax, because it's so unfriendly to grep. If you consistently avoid it and the 1.9 destructuring syntax, you can reliably grep for `task/title` and find all the usages of that keyword. 

It's less of a problem lately. When I want to find namespaced keywords I mostly use `find-references`. But I still do a certain amount of grepping in the terminal, so my preference is still to avoid map namespaces.

Reply:
No problem, as you mentioned I most of the time use find-references, but I can see how convenient for other users it can be.